docs/src/configuration/app/app-reference.md

```diff
@@ -252,7 +252,7 @@ The following table presents possible properties for each location:
 
 | Name                | Type                                                 | Default   | Description |
 | ------------------- | ---------------------------------------------------- | --------- | ----------- |
-| `root`              | `string`                                             |           | The directory to serve static assets for this location relative to the app root. |
+| `root`              | `string`                                             |           | The directory to serve static assets for this location relative to the app root. Must be an actual directory and not just the app root. |
```

Comment:
```suggestion
| `root`              | `string`                                             |           | The directory to serve static assets for this location relative to the app root. Must be an actual directory and not just the app's root folder. |
```

Reply:
Great idea for a change! I wouldn't change only the second instance of it. So I'll change the others.